### DISCUSSION TITLE
Adding Parameters

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#2591] [Q&A] [answered]
I wanted to add lithium plating parameters to the Chen2020 parameter set. Please provide any tutorial or sample code regarding this.

**Comment by brosaplanella** [comment, marked as answer]
You can update the existing Chen2020 dictionaries with your own plating parameters. The [parameterization notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/parameterization/parameterization.ipynb) has useful information on how to do this.

**Reply by kumaryash7** [reply]
We want to implement our own lithium plating kinetics by creating our custom model, but we would like to know the plating kinetics used in the pre-defined DFN model.

**Reply by brosaplanella** [reply]
The kinetics used by default are those in OKane 2022 paper (see there parameter set [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium_ion/OKane2022.py)).
